### PR TITLE
[RFC][UN-6614] Improvements to uniq

### PR DESCRIPTION
**Pull request by mmelo**
Needs uniplaces-uniq tag

Refers to: https://uniplaces.atlassian.net/browse/UN-6614

Added uni-alert component and made some minor style and behaviour fixes

Improvements to ember-cli-uniq
- [x] uni-button: add actions to be passed to button
- [ ] uni-search/combo-box/avenida (?)
- [x] Literalize uni-title (and subordinates) templating
- [x] Aggregate subtitles and subheadings
- [x] Add uni-subtitle component (class="uni-subtitle")
- [x] Add subtitle to uni-title
- [x] Create new uni-price component (inherit from uni-input-range) -- https://github.com/uniplaces/ember-cli-uniq/pull/13
- [x] Create uni-calendar component -- https://github.com/uniplaces/ember-cli-uniq/pull/14

Improvements to UNIQ
- [x] Add grid-component to agilize use of columns and so
- [x] Add grid-component to agilize use of columns and so
- [x] Aggregate subtitles and subheadings
- [x] Create input price styles
- [ ] uni-calendar styles (already done, is just incorporating that is missing)

### CODE REVIEW

File: .vscodeignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+dist/**
+tmp/**
+build/**
+cache/**
+node_modules/**
+bower_components/**
+.sass-cache/**
+connect.lock/**
+coverage/*/**
+libpeerconnection.log
```
Comment (raffo0707) [inline .vscodeignore:10]: new line

File: app/components/uni-subtitle.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export { default } from 'ember-cli-uniq/components/uni-subtitle';
```
Comment (raffo0707) [inline app/components/uni-subtitle.js]: new line

File: app/components/uni-alert.js [hunk 3]
```diff
@@ -0,0 +1 @@
+export { default } from 'ember-cli-uniq/components/uni-alert';
```
Comment (raffo0707) [inline app/components/uni-alert.js]: new line

File: jsconfig.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{"compilerOptions":{"target":"es6","experimentalDecorators":true},"exclude":["node_modules","bower_components","tmp","vendor",".git","dist"]}
```
Comment (raffo0707) [inline jsconfig.json]: new line

File: public/assets/images/icons/icon-warning.svg [hunk 5]
```diff
@@ -0,0 +1,2 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="22px" height="22px" viewBox="0 0 22 22">
+<path fillrule="evenodd" d="M 11 22C 4.93 22-0 17.07-0 11-0 4.93 4.93 0 11 0 17.07 0 22 4.93 22 11 22 17.07 17.07 22 11 22ZM 11 1.76C 5.91 1.76 1.76 5.91 1.76 11 1.76 16.09 5.91 20.24 11 20.24 16.09 20.24 20.24 16.09 20.24 11 20.24 5.91 16.09 1.76 11 1.76ZM 11 18.04C 10.27 18.04 9.68 17.45 9.68 16.72 9.68 15.99 10.27 15.4 11 15.4 11.73 15.4 12.32 15.99 12.32 16.72 12.32 17.45 11.73 18.04 11 18.04ZM 11 13.64C 10.51 13.64 10.12 13.25 10.12 12.76 10.12 12.76 10.12 4.4 10.12 4.4 10.12 3.91 10.51 3.52 11 3.52 11.49 3.52 11.88 3.91 11.88 4.4 11.88 4.4 11.88 12.76 11.88 12.76 11.88 13.25 11.49 13.64 11 13.64Z" fill="#1d2022"></path></svg>
```
Comment (raffo0707) [inline public/assets/images/icons/icon-warning.svg:2]: new line, check your configurations, editors usually have an option to put an empty line at the end if it's missing

**Review by raffo0707 — COMMENTED** [review]
remove typings/ember/ember.d.ts

File: addon/templates/components/uni-step-card.hbs [hunk 6]
```diff
@@ -1,7 +1,7 @@
 <div class="uni-step-card__step">{{step}}</div>
 
-<div class="uni-step-card__title">{{title}}</div>
-<div class="uni-step-card__hint">{{hint}}</div>
+<div class="uni-step-card__title">{{{title}}}</div>
+<div class="uni-step-card__hint">{{{title}}}</div>
```
Comment (AndreJoaquim) [inline addon/templates/components/uni-step-card.hbs]: should be hint

File: addon/templates/components/uni-subtitle.hbs [hunk 7]
```diff
@@ -0,0 +1 @@
+{{subtitle}}
```
Comment (AndreJoaquim) [inline addon/templates/components/uni-subtitle.hbs]: we might need to inject some HTML here... :D

change to `{{{subtitle}}}`

File: addon/templates/components/uni-title.hbs [hunk 8]
```diff
@@ -1 +1,4 @@
-{{title}}
+{{{title}}}
+{{#if subtitle}}
+    <p class="uni-title__subtitle">{{{subtitle}}}</p>
```
Comment (AndreJoaquim) [inline addon/templates/components/uni-title.hbs]: Use  {{uni-subtitle subtitle=subtitle}}

File: app/components/uni-alert.js (hunk 3 above)
Comment (AndreJoaquim) [inline app/components/uni-alert.js]: newline

File: app/components/uni-subtitle.js (hunk 2 above)
Comment (AndreJoaquim) [inline app/components/uni-subtitle.js]: newline

File: jsconfig.json (hunk 4 above)
Comment (AndreJoaquim) [inline jsconfig.json]: why do we need this?

File: public/assets/images/icons/icon-warning.svg (hunk 5 above)
Comment (AndreJoaquim) [inline public/assets/images/icons/icon-warning.svg:2]: newline

**Review by AndreJoaquim — CHANGES_REQUESTED** [review]

**Comment by AndreJoaquim** [comment]
As raffa said, I don't know what is that ember.d.ts but probably we don't need it :)

File: .vscodeignore (hunk 1 above)
Comment (montoias) [reply .vscodeignore:10]: why do we need  .vscodeignore ? remove this file.

File: jsconfig.json (hunk 4 above)
Comment (montoias) [reply jsconfig.json]: remove this, this is VS generated files

**Review by montoias — CHANGES_REQUESTED** [review]

**Comment by montoias** [comment]
@AndreJoaquim it's the VScode editor. Remove all of that files that I mentioned

File: public/assets/images/icons/icon-warning.svg (hunk 5 above)
Comment (mmelo) [reply public/assets/images/icons/icon-warning.svg:2]: svg's will be dealt with on build so it does not impact anything right now.

File: app/components/uni-alert.js (hunk 3 above)
Comment (mmelo) [reply app/components/uni-alert.js]: done

File: public/assets/images/icons/icon-warning.svg (hunk 5 above)
Comment (mmelo) [reply public/assets/images/icons/icon-warning.svg:2]: svg's will be dealt with on build, so this has no impact right now.

File: app/components/uni-alert.js (hunk 3 above)
Comment (mmelo) [reply app/components/uni-alert.js]: done

File: app/components/uni-subtitle.js (hunk 2 above)
Comment (mmelo) [reply app/components/uni-subtitle.js]: done

**Comment by montoias** [comment]
reviewed

**Comment by AndreJoaquim** [comment]
Remove .DS_Store file

**Pull request merged**